### PR TITLE
Update datahub-executor docs

### PR DESCRIPTION
**Pull request by noggi**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

@jjoyce0510

### CODE REVIEW

**Review by jjoyce0510 — APPROVED** [review]
Looks great! 

Do we need the account id for the customer to grant them access to the image?

**Review by jjoyce0510 — CHANGES_REQUESTED** [review]
Actually one more change required - Can we generalize this doc beyond "ingestion" since it now can handle both ingestion and Monitoring?

We should probably place a quick note in the introduction that it supports both running ingestion and running Acryl Observe related assertion monitors.

File: docs/managed-datahub/operator-guide/setting-up-remote-ingestion-executor-on-aws.md [hunk 1]
```diff
@@ -32,17 +32,14 @@ To accommodate these cases, Acryl supports configuring a remote ingestion execut
...
 
-2. **Provision an Acryl Executor** (ECS)**:** Acryl team will provide a [Cloudformation Template](https://github.com/acryldata/datahub-cloudformation/blob/master/Ingestion/templates/python.ecs.template.yaml) that you can run to provision an ECS cluster with a single remote ingestion task. It will also provision an AWS role for the task which grants the permissions necessary to read and delete from the private SQS queue created for you, along with reading the secrets you've specified. At minimum, the template requires the following parameters:
+1. **Provision an Acryl Executor** (ECS)**:** Acryl team will provide a [Cloudformation Template](https://github.com/acryldata/datahub-cloudformation/blob/master/Ingestion/templates/python.ecs.template.yaml) that you can run to provision an ECS cluster with a single remote ingestion task. It will also provision an AWS role for the task which grants the permissions necessary to read and delete from the private SQS queue created for you, along with reading the secrets you've specified. At minimum, the template requires the following parameters:
```
Comment (pedro93) [inline docs/managed-datahub/operator-guide/setting-up-remote-ingestion-executor-on-aws.md:36]: Can you add a reference to our helm chart based deployment?

**Review by pedro93 — COMMENTED** [review]
I think this doc needs to be updated for customers who already have remote executor v1 and are transitioning to v2

File: docs/managed-datahub/operator-guide/setting-up-remote-ingestion-executor-on-aws.md (hunk 1 above)
Comment (noggi) [reply docs/managed-datahub/operator-guide/setting-up-remote-ingestion-executor-on-aws.md]: added

**Comment by noggi** [comment]
> Looks great!
> 
> Do we need the account id for the customer to grant them access to the image?

Yeah, I accidentally removed the entire line, although I only meant to amend it a bit. Fixed.

**Comment by noggi** [comment]
> I think this doc needs to be updated for customers who already have remote executor v1 and are transitioning to v2

Added a note with quick migration steps.

**Review by jjoyce0510 — APPROVED** [review]

**Pull request merged**